string msgType
int64 questionNumOrPart
string questionType
string robotSpeech
string otherInfo
